davdwhyte87/iReporter
# # 162412382 Chore: add database

**What does this PR do?**
This pull request adds PostgreSQL database to the api

**Description of task to be completed?**
- add necessary packages for postgresql 
- configure the database 
- replace the in memory database with the PostgreSQL database

**How can this be manually tested?**
- clone the repository
- cd into the project folder
- run npm i
- run npm test to check that the new database system works

**Relevant pivotal tracker stories?**
` 162412382`
